arcsi/api/utils.py

```diff
@@ -122,6 +126,19 @@ def broadcast_audio(
             return True
     return False
 
+
+def delete_file(archive_base, archive_idx, archive_name):
```

Comment:
I added this function and the safe join changes into the other PR.

Reply:
Great!